src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private final Random random = new Random();
+
     public String getRandomColor() {
-        return null;
+        Color color = Color.values()[random.nextInt(Color.values().length)];
+        return color.name();
```

Comment:
It's better to store the result of `Color.values()` in a local variable before the loop to avoid calling it multiple times which can impact performance. This is because `values()` creates a new array each time it's called.